jest/react-native-device-info-mock.js

```diff
@@ -37,6 +37,8 @@ const diMock = {
   getDeviceName: stringFnAsync(),
   getDeviceNameSync: stringFnSync(),
   getDeviceToken: stringFnAsync(),
+  getDeviceType: stringFnAsync(),
```

Comment:
This duplicates the existing one, but the existing `getDeviceType` needs to change to `stringFnAsync()`

Reply:
gah! this is what I get for reviewing before the tests complete!